Objects/typevarobject.c

```diff
@@ -59,6 +59,9 @@ typedef struct {
 
 #include "clinic/typevarobject.c.h"
 
+#define typevartuple_iter unpack_iter
```

Comment:
I don't think we need these defines, just use `unpack_iter` directly.

Reply:
This was a suggestion from @ZeroIntensity to keep the naming more consistent. I can remove them though